homeassistant/components/light/tplink.py

```diff
@@ -71,11 +75,28 @@ def __init__(self, smartbulb: 'SmartBulb', name) -> None:  # noqa: F821
         self._max_mireds = None
         self._emeter_params = {}
 
+    @property
+    def unique_id(self):
+        """Return a unique ID."""
+        return self._unique_id
+
     @property
     def name(self):
-        """Return the name of the Smart Bulb, if any."""
+        """Return the name of the Smart Bulb."""
         return self._name
 
+    @property
+    def device_info(self):
+        """Return information about the device."""
+        return {
+            'identifiers': {
+                (TPLINK_DOMAIN, self._unique_id)
```

Comment:
Since it's a mac address, it's not `TPLINK_DOMAIN` but `CONNECTION_NETWORK_MAC`. And not an identifier but a connection.

Reply:
Just like the example in the docs https://developers.home-assistant.io/docs/en/device_registry_index.html#defining-devices